releases/v0.23.md

```diff
@@ -0,0 +1,182 @@
+# v0.23 🚀
+
+This document contains all release notes pertaining to the `v0.23.x` releases of
+Vela.
+
+## For Vela Administrators
+
+- When upgrading from `v0.23`, please take note of the [migration information](/migrations/v0.23/README.md)
+
+## v0.23.0
+
+### 📣 Highlights
+
+
+### 💥 Breaking Changes
+
+- (server) enhance(api/scm)!: return affected repos in API response and do not assume 404 on GetRepo error [#1015](https://github.com/go-vela/server/commit/2b58510bc4b590cb8ef550ac1774e18bba8e2f37) - thanks [@ecrupper](https://github.com/ecrupper)!
+- (server) enhance(secrets)!: use the same allow_events system as repos for secrets [#1033](https://github.com/go-vela/server/commit/78eca514b01d626405cc07887eaee1c4e8830352) - thanks [@ecrupper](https://github.com/ecrupper)!
+- (server) feat(api/database)!: store deployment record in database for Vela-targeted deployments [#1031](https://github.com/go-vela/server/commit/a7cd07fe43cd7e5e0e79f7e1ddb6ca055d518d8a) - thanks [@claire1618](https://github.com/claire1618)!
+- (server) feat(repo)!: support allow_events struct [#1023](https://github.com/go-vela/server/commit/6fcde8652f63e317137a63cb4f01586bbbbc49c2) - thanks [@ecrupper](https://github.com/ecrupper)!
+- (server) feat(webhook)!: support build approval based on repository settings [#1016](https://github.com/go-vela/server/commit/5fc317f98d5c56583cd19ecfd2e714747d5dd3cf) - thanks [@ecrupper](https://github.com/ecrupper)!
+- (types) feat(repo)!: change `allow_<event>` fields to an `AllowEvents` struct + DB use integer masking [#314](https://github.com/go-vela/types/commit/23c15858da2a56259b0cf6c2cf77aeb9e24d3f95) - thanks [@ecrupper](https://github.com/ecrupper)!
+
+#### Users:
+- `v0.23.x` introduces a new support build approval based on repository settings. Please take note of the [build approval information](/migrations/v0.23/build-approval.md).
+- `v0.23.x` introduces a new `deployments` table that will be created automatically by default. However, if you are running the `server` component with the `VELA_DATABASE_SKIP_CREATION` set to `true`, you will need to manually create this table.
```

Comment:
This sentence should be in the `For Vela Administrators` section, but the bullet below should stay here in `Users`

Reply:
Done